app/views/home/how_government_works.html.erb

```diff
@@ -1,534 +1,489 @@
-        heading_level: 2,
-        margin_bottom: 3,
-      } %>
-      <div class="halves-width-section history">
-        <div class="one-half">
-          <div class="inner-block">
-            <%= image_tag 'how-gov-works/Churchill_01.jpg', alt: 'History of government' %>
-          </div>
-        </div>
-        <div class="one-half">
-          <div class="inner-block">
-            <p class="govuk-body">Britain has one of the oldest governments in the world. Find out more about how it has worked and who has shaped it in the <%= link_to "history", "/government/history/", class: "govuk-link" %> section.</p>
+        <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+          <li>look at what the government is doing</li>
+          <li>debate issues and pass new laws</li>
+          <li>set taxes</li>
+        </ul>
 
-            <p class="govuk-body">Read about past Prime Ministers, Chancellors and Foreign Secretaries in <%= link_to "notable people", "/government/history#notable-people", class: "govuk-link" %>. Learn more about historic <%= link_to "government buildings", "/government/history#government-buildings", class: "govuk-link" %> on Whitehall and around the UK.</p>
+        <p class="govuk-body">
+          <%= link_to "Find out about how Parliament works", "https://www.parliament.uk/", class: "govuk-link", rel: "external" %>
+        </p>
 
-            <p class="govuk-body">You can also find links to <%= link_to "historical research", "/government/history#historical-research", class: "govuk-link" %>, documents and records.</p>
-          </div>
-        </div>
-      </div>
+        <p class="govuk-body">Read The Official Report at <%= link_to "Hansard", "http://www.parliament.uk/business/publications/", class: "govuk-link", rel: "external" %></p>
+      </section>
+
+      <section>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "History of government",
+          id: "history-uk-government",
+          font_size: "l",
+          margin_bottom: 4,
+        } %>
+
+        <p class="govuk-body">Britain has one of the oldest governments in the world. Find out more about how it has worked and who has shaped it in the <%= link_to "history", "/government/history/", class: "govuk-link" %> section.</p>
+        <p class="govuk-body">Read about past Prime Ministers, Chancellors and Foreign Secretaries in <%= link_to "notable people", "/government/history#notable-people", class: "govuk-link" %>. Learn more about historic <%= link_to "government buildings", "/government/history#government-buildings", class: "govuk-link" %> on Whitehall and around the UK.</p>
+        <p class="govuk-body govuk-!-margin-bottom-8">You can also find links to <%= link_to "historical research", "/government/history#historical-research", class: "govuk-link" %>, documents and records.</p>
+        
+        <%= image_tag 'how-gov-works/Churchill_01.jpg', alt: 'History of government' %>
```

Comment:
In the context of this page, I'm not sure about image use in general. However, keeping this in scope of this PR - I think the alt of this needs to change to convey the meaning of this image.  I would go with something like "Winston Churchill" or even "Sir Winston Leonard Spencer Churchill"

Reply:
I'd say in this case it's entirely decorative and not conveying any information whatsoever, so set the alt text to an empty string.

Reply:
TL;DR - on review I agree. Let's remove alt for all these images on this page.

Reply:
Provided empty) alt text for the following images -
```
<%= image_tag @prime_minister.image_url(:s465), { alt: '' } %>
<%= image_tag 'how-gov-works/10_Downing_Street.jpg', alt: '' %>
<%= image_tag 'how-gov-works/Cabinet_01.jpg', alt: '' %>
<%= image_tag 'how-gov-works/Churchill_01.jpg', alt: '' %>
```